virtualdns.go

```diff
@@ -0,0 +1,112 @@
+package cloudflare
+
+import (
+    "encoding/json"
+    "github.com/pkg/errors"
+)
+
+/*
+Create a Virtual DNS cluster
```

Comment:
Can you change this to mirror the (golint approved) comment style? Refer to https://github.com/cloudflare/cloudflare-go/blob/master/dns.go#L10-L13